.spelling

```diff
@@ -77,6 +77,7 @@ init
 labeled
  - demos/Azure/README.md
 alpha.11
```

Comment:
`alpha.11` shouldn't be needed now, otherwise it means we left some docs not updated 😄